additional dependencies required for yum based distros

stebo85 opened the pull request:
## Description of the Pull Request (PR):

I got the error

```
glib-2.0 headers are required to build conmon.
```

following the original instructions. I had to install the additional dependencies to get around the issue.

WestleyK commented:
Notice in the [signNverify](https://github.com/sylabs/singularity-userdocs/blob/a36a5eef1283289c789c4289c42f9f32b4f2947e/signNverify.rst) page, all of the key commands have the `s` in it, (`keys`, vs `key`). For 3.0.3 thats fine, but for 3.1 that need to change to `key`.

devprincess commented on signNverify.rst:
```diff
@@ -117,6 +117,29 @@ again like so.
+Searching for keys
+==================
+
+Singularity allows you to search the keystore for plublic keys. You can search for names,
```
typo on word "public" here

GwendolynK commented on signNverify.rst:
```diff
@@ -117,6 +117,29 @@ again like so.
+==================
+
+Singularity allows you to search the keystore for plublic keys. You can search for names,
+email, and key IDs (fingerprints). When searching for a fingerprint, you need to use ``0x``
```
'email' should be plural ('emails'). 

Also, because 'key IDs' are called 'fingerprints' in the following sentence, it doesn't make sense to have 'fingerprints' in parentheses. I suggest changing this either to:
"....emails, and fingerprints (key IDs). ..."
_or_
"...emails, and key IDs, which are also called fingerprints. ..."

devprincess commented:
@WestleyK is the search still working with the `0x` at the beginning? if that's so then we can merge, otherwise I'll wait the update for the examples :+1:

phpHavok commented:
@WestleyK I updated the keys -> key thing in #144 , but my update doesn't include the changes in this PR.

devprincess commented:
@phpHavok @WestleyK don't worry guys, I'll update these commands on this section :+1: thanks

stebo85 committed:
additional dependencies required

I got the error

```
glib-2.0 headers are required to build conmon.
```

following the original instructions. I had to install the additional dependencies to get around the issue.

dtrudg commented:
Closing as addressing in #148 which also updates the debian deps.

`glibc-devel` is brought in by the 'Development Tools' group, so shouldn't be required explicitly.
